tests/MongoDB.Driver.Tests/Specifications/client-side-encryption/prose-tests/ClientEncryptionProseTests.cs

```diff
@@ -2234,11 +2342,28 @@ public DisposableEnvironmentVariable(string name, string value)
                 _name = name;
                 _previousValue = Environment.GetEnvironmentVariable(name);
                 Environment.SetEnvironmentVariable(name, value);
+                Ensure.That(Environment.GetEnvironmentVariable(_name) == value, "New env variable value has not been set.");
```

Comment:
We should include `_name` in the error message.

Reply:
this part is removed